### PR TITLE
Refactor customized emails

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

**Comment by xdamman** [comment]
This assumes that there is an opensource version of every template no? Would it fall back to the general template if such custom email for a particular template doesn’t exist?

**Comment by znarf** [comment]
@xdamman In this case, it's just for `collective.created`.

File: server/lib/email.js [hunk 1]
```diff
@@ -226,6 +226,7 @@ const isWhitelistedDomain = email => {
  */
 const generateEmailFromTemplate = (template, recipient, data = {}, options = {}) => {
   const slug = get(options, 'collective.slug') || get(data, 'collective.slug') || 'undefined';
+  const hostSlug = get(data, 'collective.slug');
```
Comment (Betree) [inline server/lib/email.js:229]: Shouldn't that be

```suggestion
  const hostSlug = get(data, 'collective.host.slug');
```

?

Comment (znarf) [reply server/lib/email.js:229]: @Betree Oops. Definitely !

**Pull request merged**